src/rtp/id.rs

```diff
@@ -189,6 +202,23 @@ impl SeqNo {
     pub(crate) fn is_max(&self) -> bool {
         self.0 == Self::MAX.0
     }
+
+    /// Get the rollover counter (ROC) value.
+    ///
+    /// ```
+    /// # use str0m::rtp::SeqNo;
+    /// // More than 2^16, thus rolled over.
+    /// let seq_no: SeqNo = 95_000.into();
+    ///
+    /// assert_eq!(seq_no.roc(), 1);
+    ///
+    /// // Is the same as shifting 16 bits.
+    /// assert_eq!(seq_no.roc(), 95_000 >> 16);
+    /// ```
+    #[inline(always)]
+    pub fn roc(&self) -> u64 {
```

Comment:
Nice, was just thinking that this method would be useful. Can you adopt in `src/rtp/srtp.rs`? too?

Reply:
Sure

Reply:
Ah, of course.